README.md

```diff
@@ -7,43 +7,49 @@ Pangenome Graph Queries in Calyx
 ================================
 
 This is a nascent project to build a DSL-to-hardware compiler using [Calyx][] to implement pangenomic graph queries in the vein of [odgi][].
-It is very much a work in progress.
 
 Getting Started
 ---------------
 
-First, clone the repo using 
-```git clone https://github.com/cucapra/pollen.git```
-Then follow the instructions below to set up `calyx` and `odgi`.
+### Installation
+
+
+#### Pollen
+
+Clone this repository using 
+```
+git clone https://github.com/cucapra/pollen.git
+```
+and then follow the instructions below to set up our dependencies, `calyx` and `odgi`.
 
-### Installing Dependencies
 
 #### Calyx
-Follow these [instructions](https://docs.calyxir.org/) to install calyx. You must complete the [first](https://docs.calyxir.org/#compiler-installation) and [third](https://docs.calyxir.org/#installing-the-command-line-driver) sections, but feel free to skip the second. We recommend using the native calyx interpreter, so once `fud` is set up, run
+
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You must complete the [first](https://docs.calyxir.org/#compiler-installation) and [third](https://docs.calyxir.org/#installing-the-command-line-driver) sections, but feel free to skip the second. We recommend using the native calyx interpreter, so once `fud check` has passed, run
```

Comment:
I think saying "once `fud check`" has passed is a bit misleading, because it is expected that many stages won't be installed and when you run `fud check`, roughly the same number of checks will 'fail' as will 'pass.' I think as long as `fud check` runs, things should be good. We could say something like
> once `fud check` has passed (it is expected that some tools will not be available), run

Reply:
Looks great!